addon/spaniel-engines/ember-spaniel-engine.js

```diff
@@ -1,5 +1,5 @@
 import Ember from 'ember';
-const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : () => {};
+const rAF = (typeof window === 'object') && typeof window.requestAnimationFrame === 'function' ? window.requestAnimationFrame : (callback) => callback();
```

Comment:
This appears to introduce an ordering change which may cause zalgo. As previously this would be invoked asynchronously, now it is sync.